specification/ai/DocumentIntelligence/models.tsp

```diff
@@ -436,6 +428,7 @@ model Error {
   innererror?: InnerError;
 }
 
+#suppress "@azure-tools/typespec-azure-core/property-name-conflict" "https://github.com/Azure/typespec-azure/issues/417"
 @doc("An object containing more specific information about the error.")
 model InnerError {
```

Comment:
Ideally, `ErrorResponse` (and `Error`, `InnerError` within) should be what's from Azure.Core 
https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-core/lib/foundations.tsp#L131
https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-core/lib/foundations.tsp#L148-L174

There seems only one diff of property in `InnerError`, from yours and Azure.Core's.

Is this used in some success response? Or it is only used in error response (e.g. 4##)? If latter, it should be relatively safe to switch.

Reply:
The diff is actually that Document Intelligence uses camelCase (succeeded, notStarted) instead of PascalCase (Succeeded, NotStarted) because DI predates the final guidelines on status casing.  I don't believe we can easily switch.

Reply:
Yes, but I think it only affect the `OperationStatus` model, but not the `Error` model?

The `OperationStatus` model defined in service typespec could still refer to `Azure.Core.Foundations.Error`?

Reply:
Sorry, I misread.  The difference is InnerError.message, which is not defined in Azure.Core.  We use this property extensively.  We can define a subclass of InnerError, but that would force users to cast, which would be very unintuitive.

We technically can share ErrorResponse.  But since we don't yet return x-ms-error-code, sharing right now would arguably be misleading.

Thanks for tracing the TypeSpec at this level of details to suggest this.